Update tcp_queue_length README.md (#8386)
DataDog/integrations-core: tcp_queue_length/README.md

### Metrics

> For each container or a pod, tcp_queue_length integration returns the connection read/write buffer which is the mostly used. For example, if a container has three TCP connections, for which their read buffers are used 40%, 25% and 80% respectively, the metric `tcp_queue.read_buffer_max_usage_pct` will return 0.8

Reviewer: ```suggestion
For each container of a pod, tcp_queue_length integration returns usage in percent of the connection read/write buffer which is the mostly used. For example, if a container has three TCP connections, for which their read buffers are used 40%, 25% and 80% respectively, the metric `tcp_queue.read_buffer_max_usage_pct` will return 0.8
```

Author: Will remove "pod" because the integration also applies to plain docker and ECS containers.

> For each container, tcp_queue_length integration returns the read/write buffer's fill percentage of the busiest TCP connection. For example, if a container has three TCP connections, for which their read buffers (i.e. receive queues) are 40%, 25% and 80% full (respectively), the metric `tcp_queue.read_buffer_max_usage_pct` will return 0.8 (80%), which is the maximum.

Reviewer 2: ```suggestion
For each container, the `tcp_queue_length` integration returns the read/write buffer's fill percentage of the busiest TCP connection. For example, if a container has three TCP connections, for which their read buffers (that is, receive queues) are 40%, 25%, and 80% full (respectively), the metric `tcp_queue.read_buffer_max_usage_pct` returns the maximum, 0.8 (80%).
```
